lit/SwiftExpr/SimpleExpressions.test

```diff
@@ -0,0 +1,71 @@
+// Test that we can define and use basic functions/expressions in the REPL.
```

Comment:
This one does pretty much everything TestREPLBasic does.  Add a simple variable assignment, and then you should be able to delete that test as well.

Reply:
Will do, thanks!